nuclia/tests/fixtures.py

```diff
@@ -3,29 +3,36 @@
 
 import pytest
 
+from nuclia import BASE_DOMAIN
 from nuclia.config import reset_config_file, set_config_file
 from nuclia.sdk.auth import NucliaAuth
 
-TESTING_KBID = "eb720a59-f879-4b23-a995-605f91c874f4"
-TESTING_KB = "https://europe-1.nuclia.cloud/api/v1/kb/" + TESTING_KBID
+if BASE_DOMAIN == "stashify.cloud":
+    TESTING_ACCOUNT_SLUG = "eric-cicd"
+    TESTING_KBID = "84379bd4-41d4-4100-86a2-9e5512675ae3"
+    TESTING_KB = "https://europe-1.stashify.cloud/api/v1/kb/" + TESTING_KBID
+else:
+    TESTING_ACCOUNT_SLUG = "nuclia"
+    TESTING_KBID = "eb720a59-f879-4b23-a995-605f91c874f4"
+    TESTING_KB = "https://europe-1.nuclia.cloud/api/v1/kb/" + TESTING_KBID
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module")
```

Comment:
out of curiosity: why did you need to change this fixture scope to module?

Reply:
If not run in isolation, the tests we do at account level are impacting the test we do at kb level (like the kb token is removed when updating the list of kbs, things like that).
Instead of patching the impacts, I though it was safer to just separate the 2 test suites.